libfreerdp/codec/h264_ffmpeg.c

```diff
@@ -231,7 +246,7 @@ static int libavcodec_compress(H264_CONTEXT* h264, BYTE** ppDstData, UINT32* pDs
 	if (!libavcodec_create_encoder(h264))
 		return -1;
 
-	av_free(sys->packet.data);
+	av_packet_unref(&sys->packet);
```

Comment:
This breaks old FFMPEG versions (e.g. ubuntu 14.04)
Do a version AVCODEC check >= 55.39.100 the new style, otherwise the old code you deleted